envoy/stream_info/BUILD

```diff
@@ -34,6 +34,7 @@ envoy_cc_library(
     hdrs = ["filter_state.h"],
     external_deps = ["abseil_optional"],
     deps = [
+        "//source/common/common:inline_map_registry_helper",
```

Comment:
ah good. Now...I think I'm going to recommend renaming "registry" to "descriptor" and "registry_helper" to "registiry" :)

WDYT?

Reply:
SGTM. Let we figure out conlustons about static definition and underlay storage first. Finally, to address this renaming.